### PR TITLE
Pr Role Create pub key in Remote Server

### PR DESCRIPTION
**Pull request by talits**
Change directories for roles
Set .pub key in remote server on authorized_keys - Role qedu-user-create

### CODE REVIEW

File: qedu-user-create/templates/authorized_keys.j2 [hunk 1]
```diff
@@ -0,0 +1,13 @@
+ssh-rsa AAAAB3NzaC1yc2EAAAADAQABAAABAQC3di5binozK2IVNPB2dwMjoN/CrF30SqfhOr2CfNTP7K9Xv1ULsWiy0GkIO1s2x37JXTJGaXSbmROrc+h4n2TrysO9fWdwDAxqKAn3E/NsgXcg7FSSEJ3m3CQhm0YYFKN6lcPrxQiK6pcQR8BHoVwu9kbsrzGIn5+sJCQq744LoQbd6A7goaiB5tqRZd9zawjRflNksSlAjHR9FpbPyIZXyD1aVcKVbnLpg72fWNddXZ2m7zFCofbeCdGAabjXBnvEh7UxUXszf8nrBSKv9fJWKynYII/3SfQ6Hd0WOOkTgjXkmJfed0YIIdSL1oTKGNvafzfSoTw3ixUTHkGwz9gl qedu@qedu-web-master
```
Comment (raulbcs) [inline qedu-user-create/templates/authorized_keys.j2]: Is this file required?

**Review by raulbcs — COMMENTED** [review]
Hi Talita,
Thanks for the modifications. 
I have just one question about the file named `qedu-user-create/templates/authorized_keys.j2`.
If not, could you please delete?

Thanks again! (:
Raul

**Comment by talits** [comment]
Oh well , I will delete

**Comment by talits** [comment]
It's ok now
